docs/v0.7/en/user-guide/write-data/prometheus.md

```diff
@@ -22,11 +22,24 @@ remote_read:
 Be sure to uncomment `basic_auth` section and replace `greptime_user(username)`, `greptime_pwd(password)` with your own username and password when you enable database authentication. Please refer to client [authentication](../clients/authentication.md).
 :::
 
-The `db` parameter in the url represents the database that we want to write data.
+The `db` parameter in the url represents the database that we want to write data. It's optional.
 By default, the database is `public`.
 If you want to write to another database, you can [create a new database](../table-management.md#create-database) 
 and replace `public` with the new database name.
 
+The `physical_table` parameter in the url represents the physical table that we want to write data. In GreptimeDB's `Metric` engine, we have a concept of physical table which specifies the table that store data physically. This parameter is optional. If you don't specify it, the default physical table name `greptime_physical_table` will be used. This physical table will be created automatically if it doesn't exist.
+
+Notice that this `physical_table` parameter only works when
+- `with_metric_engine` is enabled in the configuration file. It's enabled by default.
+- There is no previous data in corresponding `db`. I.e., those metrics to be written are first seen to that `db`.
```

Comment:
confused about what does this mean...what is `previous data`? what is `first seen`?

Reply:
I wanna express that "the table didn't exist before", where the "table" references the metric to be ingested. Could you please help me to word it?